bindings/go/README.md

```diff
@@ -52,6 +52,8 @@ In order to build:
  1. [Build Motr](../../doc/Quick-Start-Guide.rst) first
    (or install pre-built motr-devel package).
  2. Install Go: `sudo yum install go`.
+ 
+Note: Go version 1.16 or above is required to build this project.
```

Comment:
Why?

Reply:
No actually. 

This table should explain much clearer.

|    Go     |  support module  |     work with code without module	             |
|--------- | ------------------  | ---------------------------------------------    |
|   1.15    |	   no	                     |	   yes					                     |
|   1.16    |	   yes                     |	   No by default, but can enabled. 	             |
|   1.17    |	   yes                     |    No. Plan support code with module only. |

For detail explanation, see [here](https://blog.golang.org/go116-module-changes).

Perhaps we could upgrade only once 1.16 become official distribution in Linux ?

Reply:
Strange. It works for me with go-1.15 too. What error do you see?

Reply:
My mistake, Go Module supported since Go 1.11, see [here](https://blog.golang.org/migrating-to-go-modules).

I recompile the changed code with Go 15.5, it compile with no issue.

I also had fixed the commit message.